ts/utils/supportAssistance.ts

```diff
@@ -170,6 +170,15 @@ export const defaultZendeskIDPayCategory: ZendeskCategory = {
   }
 };
 
+export const defaultZendeskBonusesategory: ZendeskCategory = {
```

Comment:
There's a typo here 👀 
```suggestion
export const defaultZendeskBonusesCategory: ZendeskCategory = {
```